### PR TITLE
[wemo] Migrate to bnd build

### PR DESCRIPTION
**Pull request by wborn**
Migrates the Wemo Binding to the bnd build.

### CODE REVIEW

File: itests/org.openhab.binding.max.tests/.project [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.openhab.binding.wemo.test</name>
-	<comment>This is the test component for Eclipse SmartHome Wemo Binding</comment>
+	<name>org.openhab.binding.max.tests</name>
```
Comment (martinvw) [inline itests/org.openhab.binding.max.tests/.project]: wemo / max?

Comment (wborn) [reply itests/org.openhab.binding.max.tests/.project]: I'll remove it. Eclipse likes to annoy me by readding project files when the projects are still open but no longer exist on the branch. :smile:

**Comment by martinvw** [comment]
@wborn is the failing build related to your changes?

**Comment by wborn** [comment]
It was unrelated @martinvw. After a rebuild this PR now also makes Travis happy.

**Pull request merged**